app/views/admin/categories/_status.html.erb

```diff
@@ -1,7 +1,7 @@
 <%= content_tag :div, class: 'loader-btn' do %>
   <% if locals[:category].enabled %>
```

Comment:
This was awesome. Thanks for sharing this one. Don't know why didn't it click to me. I have many places to improve this thing.